src/web/screens/dashboard.tsx

```diff
@@ -16,7 +16,6 @@ import { ParticipantInformationRoute } from './participantInformation';
 import { PortalRoute } from './routeUtils';
 import { SharingPermissionsRoute } from './sharingPermissions';
 import { TeamMembersRoute } from './teamMembers';
-import { TermsOfServiceRoute } from './termsOfService';
```

Comment:
This route/screen is no longer needed because terms and conditions is now a dialog that pops up when the user logs in for the first time